gcloud-java-compute/src/main/java/com/google/gcloud/compute/DiskType.java

```diff
@@ -0,0 +1,227 @@
+ * {@code pd-ssd} or {@code pd-standard}.
+ */
+public final class DiskType implements Serializable {
+
+  static final Function<com.google.api.services.compute.model.DiskType, DiskType> FROM_PB_FUNCTION =
+      new Function<com.google.api.services.compute.model.DiskType, DiskType>() {
+        @Override
+        public DiskType apply(com.google.api.services.compute.model.DiskType pb) {
+          return DiskType.fromPb(pb);
+        }
+      };
+  static final Function<DiskType, com.google.api.services.compute.model.DiskType> TO_PB_FUNCTION =
+      new Function<DiskType, com.google.api.services.compute.model.DiskType>() {
+        @Override
+        public com.google.api.services.compute.model.DiskType apply(DiskType diskType) {
+          return diskType.toPb();
+        }
+      };
+
+  private static final long serialVersionUID = -944042261695072026L;
+
+  private final Long id;
+  private final DiskTypeId diskTypeId;
+  private final String creationTimestamp;
+  private final String description;
+  private final String validDiskSize;
+  private final String selfLink;
+  private final Long defaultDiskSizeGb;
+
+  static final class Builder {
+
+    private Long id;
+    private DiskTypeId diskTypeId;
+    private String creationTimestamp;
+    private String description;
+    private String validDiskSize;
+    private String selfLink;
+    private Long defaultDiskSizeGb;
+
+    private Builder() {}
+
+    Builder id(Long id) {
+      this.id = id;
+      return this;
+    }
+
+    Builder creationTimestamp(String creationTimestamp) {
+      this.creationTimestamp = creationTimestamp;
+      return this;
+    }
+
+    Builder diskTypeId(DiskTypeId diskTypeId) {
+      this.diskTypeId = diskTypeId;
+      return this;
+    }
+
+    Builder description(String description) {
+      this.description = description;
+      return this;
+    }
+
+    Builder validDiskSize(String validDiskSize) {
+      this.validDiskSize = validDiskSize;
+      return this;
+    }
+
+    Builder selfLink(String selfLink) {
+      this.selfLink = selfLink;
+      return this;
+    }
+
+    Builder defaultDiskSizeGb(Long defaultDiskSizeGb) {
+      this.defaultDiskSizeGb = defaultDiskSizeGb;
+      return this;
+    }
+
+    /**
+     * Creates a {@code DiskType} object.
+     */
+    DiskType build() {
+      return new DiskType(this);
+    }
+  }
+
+  private DiskType(Builder builder) {
+    this.id = builder.id;
+    this.creationTimestamp = builder.creationTimestamp;
+    this.diskTypeId = builder.diskTypeId;
+    this.description = builder.description;
+    this.validDiskSize = builder.validDiskSize;
+    this.selfLink = builder.selfLink;
+    this.defaultDiskSizeGb = builder.defaultDiskSizeGb;
+  }
+
+  /**
+   * Returns the creation timestamp in RFC3339 text format.
+   *
+   * @see <a href="https://www.ietf.org/rfc/rfc3339.txt">RFC3339</a>
+   */
+  public String creationTimestamp() {
```

Comment:
Do you think it'd be more helpful to users if we converted this into something like epoch milliseconds?